src/pal/src/CMakeLists.txt

```diff
@@ -70,6 +63,16 @@ elseif(PAL_CMAKE_PLATFORM_ARCH_ARM64)
   add_definitions(-D_WIN64=1)
 endif()
 
+if(CMAKE_SYSTEM_NAME STREQUAL Linux AND NOT CLR_CMAKE_PLATFORM_ALPINE_LINUX)
```

Comment:
Yes, we don't support that on other OS-es like FreeBSD, NetBSD or OSX. They implement this differently.